libcxx/test/std/re/re.iter/re.regiter/re.regiter.incr/post.pass.cpp

```diff
@@ -114,5 +114,53 @@ int main(int, char**)
         assert(i == e);
     }
 
+  {
+    // Check that we correctly adjust the match prefix when dealing with zero-length matches -- this is explicitly
```

Comment:
Please let me know if you think I'm writing too much detail for this single specific issue (here and/or in the test file).